.github/renovate.json

```
@@ -20,6 +20,23 @@
 		{
 			"paths": [ "packages/codesniffer/composer.json" ],
 			"rangeStrategy": "replace"
+		},
+		{
+			"groupName": "Instant Search Dependency Updates",
+			"packageNames": [
+				"cache",
+				"preact",
+				"progress-event",
+				"q-flat",
+				"qss",
+				"strip",
+				"uuid",
+				"@testing-library/preact"
+			],
+			"reviewers": "team:@Automattic/jetpack-search",
```

The docs aren't entirely clear, but it seems the correct syntax omits the "@Automattic/" from the team name.
```suggestion
			"reviewers": "team:jetpack-search",
```

It uses [this API](https://docs.github.com/en/rest/reference/pulls#request-reviewers-for-a-pull-request) ([code](https://github.com/renovatebot/renovate/blob/7f75e5768bc34bb45245336c9fa0c6eff1ebcdc6/lib/platform/github/index.ts#L1175-L1200)), which says it takes "team slugs", which seem to not include the organization based on various examples [here](https://docs.github.com/en/rest/reference/teams).